internal/common/dsschema/pagination_schema.go

```diff
@@ -6,12 +6,9 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
 )
 
-func PaginatedDSSchema(arguments, resultAttributes map[string]schema.Attribute) schema.Schema {
+func PaginatedDSSchema(arguments, resultAttributes map[string]schema.Attribute, skipID bool) schema.Schema {
 	result := schema.Schema{
 		Attributes: map[string]schema.Attribute{
-			"id": schema.StringAttribute{
```

Comment:
@AgustinBettati do you have more context of why we include an `id` attribute?
I want to avoid it to skip the `id.UniqueId()` import from sdv v2 and since it is not necessary in TPF.

Reply:
Good question, I have a memory there was an error being returned from `terraform-plugin-testing` in acceptance tests if the data source did not have an id present. Would double check on this, and if no issues are faced I agree with the approach you have taken.

Reply:
Found more context here: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/atlasuser/data_source_atlas_users.go#L55

Reply:
Nice. Looks like it is no longer needed: [PR](https://github.com/hashicorp/terraform-plugin-testing/pull/164)